src/sfCommand.ts

```diff
@@ -476,49 +472,6 @@ export abstract class SfCommand<T> extends Command {
     throw err;
   }
 
-  /**
-   * Format errors and actions for human consumption. Adds 'Error (<ErrorCode>):',
-   * When there are actions, we add 'Try this:' in blue
-   * followed by each action in red on its own line.
-   * If Error.code is present it is output last in parentheses
-   *
-   * @returns {string} Returns decorated messages.
-   */
-  protected formatError(error: SfCommand.Error): string {
```

Comment:
Removing this method could break commands that extend SfCommand.

Reply:
we don't have any

https://github.com/search?q=%28org%3Asalesforcecli+OR+org%3Aforcedotcom%29+formatError+language%3ATypeScript++NOT+is%3Aarchived&type=code

I couldn't find any outside of our repos, either.